redis-persistence/src/main/java/com/netflix/conductor/dao/dynomite/RedisExecutionDAO.java

```diff
@@ -115,6 +116,7 @@ public RedisExecutionDAO(DynoProxy dynoClient, ObjectMapper objectMapper, Config
 		return tasks;
 	}
 
+	@FaultInjectionInterceptor
```

Comment:
Since this annotation and the interceptor are used only for testing, can this abstracted out from the execution path. Maybe through some combination of Spy and Mock?